### DISCUSSION TITLE
Face recognition with thermal camera

### DISCUSSION THREAD

**Discussion by Hvaheterdu** [opendr-eu/opendr#422] [Problems and questions] [answered]
Hello!

I use TIAGo from Pal Robotics and OpenDR for my master's thesis at the University of Oslo. I plan to conduct my experiment very soon and have added a blurring feature to the face recognition node in the OpenDR repo. I am using an Optris PI series thermal camera. 

Has the face recognition node been tested with a thermal camera, and if it has, what was the result?

Thank you for creating an excellent toolkit!

In advance, thank you.

Sincerely,
Burhan Sarfraz

**Comment by passalis** [comment, marked as answer]
There is currently no provision for using thermal cameras in the toolkit.

**Comment by Hvaheterdu** [comment]
Hello, and thank you for your answer. I can confirm and say that the face recognition node worked with the thermal camera.
